sashimi/scanning.py

```diff
@@ -490,3 +490,38 @@ def run(self):
                 self.parameters = deepcopy(
                     scanloop.parameters
                 )  # set the parameters to the last ones received in the loop
+
+
+
+
+class MockScanner(Process):
```

Comment:
The mock classes are there for testing, a full solution for this requires a different version, perhaps `DisabledScanner`. Anyway I will work on it in the context of #79, you can leave it like this for the current PR.

Reply:
Will there be a disabling of the GUI for Mock/Disabled versions as well?